.github/workflows/ci.yml

```diff
@@ -8,6 +8,9 @@ on:
     branches:
     - main
 
+env:
+  CHECKMK_VERSION: "2.4.0p12"
```

Comment:
do we need to add some annotation here for DependaBot to recognize this as a docker image?